test/e2e/common/configmap_volume.go

```diff
@@ -637,8 +634,14 @@ func doConfigMapE2EWithoutMappings(f *framework.Framework, uid, fsGroup int64, d
 		},
 	}
 
-	if userID != 0 {
-		pod.Spec.SecurityContext.RunAsUser = &userID
+	if asUser {
+		if framework.NodeOSDistroIs("windows") {
+			userName := "ContainerUser"
```

Comment:
Making "ContainerUser" a constant somewhere and explain why this is used.

Reply:
Done